bedrock/settings/static_media.py

```diff
@@ -970,6 +970,13 @@
         ),
         'output_filename': 'js/contribute-form-bundle.js',
     },
+    'contribute-friends': {
+        'source_filenames': (
+            'js/libs/jquery-1.7.1.min.js',
```

Comment:
Does it specifically need to be this older version of jQuery? We've updated to 1.11.0 elsewhere, and I think 1.7 has some deprecated stuff that might throw errors.